provision/upload/config/sftpgo.yaml

```diff
@@ -55,7 +55,7 @@ sftpd:
 data_provider:
   driver: postgresql
   # from config/settings.default.yml
-  name: baw_local_development
+  #name:
```

Comment:
Question: Why is this commented out? Why not just remove the property of give it a semantic name?

Reply:
To signal intentional omission. I'll add a comment to make it clearer